experimental/packages/web-common/package.json

```
@@ -0,0 +1,104 @@
+{
+  "name": "@opentelemetry/web-common",
+  "version": "0.53.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Common utilities for web instrumentation",
+  "author": "OpenTelemetry Authors",
+  "homepage": "https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/web-common",
+  "license": "Apache-2.0",
+  "main": "build/src/index.js",
+  "module": "build/esm/index.js",
+  "esnext": "build/esnext/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-telemetry/opentelemetry-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
+  },
+  "engines": {
+    "node": ">=14"
+  },
+  "scripts": {
+    "clean": "tsc --build --clean tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../",
+    "codecov:browser": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../../",
+    "compile": "tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "peer-api-check": "node ../../../scripts/peer-api-check.js",
+    "prepublishOnly": "npm run compile",
+    "precompile": "cross-var lerna run version --scope $npm_package_name --include-dependencies",
+    "prewatch": "npm run precompile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "tdd:browser": "karma start",
+    "test": "nyc mocha test/**/*.test.ts",
+    "test:browser": "karma start --single-run",
+    "version": "node ../../../scripts/version-update.js",
+    "watch": "tsc --build --watch tsconfig.json tsconfig.esm.json tsconfig.esnext.json",
+    "align-api-deps": "node ../../../scripts/align-api-deps.js"
+  },
+  "keywords": [
+    "opentelemetry",
+    "web",
+    "tracing",
+    "profiling",
+    "stats",
+    "monitoring"
+  ],
+  "files": [
+    "build/esm/**/*.js",
+    "build/esm/**/*.js.map",
+    "build/esm/**/*.d.ts",
+    "build/esnext/**/*.js",
+    "build/esnext/**/*.js.map",
+    "build/esnext/**/*.d.ts",
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "peerDependencies": {
+    "@opentelemetry/api": ">=1.4.0 <1.10.0"
+  },
+  "dependencies": {
+    "@opentelemetry/sdk-logs": "^0.53.0",
+    "@opentelemetry/sdk-trace-base": "^1.26.0",
+    "@opentelemetry/semantic-conventions": "^1.27.0"
```

Since `@opentelemetry/semantic-conventions/incubating` is used, we need to pin to the exact version (any exports there may break in minor bumps)
```suggestion
    "@opentelemetry/semantic-conventions": "1.27.0"
```
or

```suggestion
    "@opentelemetry/semantic-conventions": "~1.27.0"
```